README.md

```diff
@@ -19,6 +19,11 @@ rustup target add wasm32-unknown-unknown
 cargo install pwasm-utils
 ```
 
+One should then run `wasm-build` agains plain Cargo-generated `.wasm` artifact (`target/wasm32-unknown-unknown/release/pwasm_tutorial_contract.wasm`) to trim and optimise it for blockchain usage.
+That is done with a single command `wasm-build --target=wasm32-unknown-unknown ./target pwasm_tutorial_contract` (it assumes the proper subfolder structure of Cargo's `target` folder automatically); that invocation results in a `target/pwasm_tutorial_contract.wasm` blockchain-ready bytecode being produced.
```

Comment:
While I can understand from where does that confusion comes from — I was even a victim of it myself! — I don't think this distinction would be very helpful at the beginning of this tutorial.

Since we're aiming more on the "get up and running with Rust for smart contracts", I think we can gloss over some details of internal implementation of our tooling. Let's just make sure that this step is clearly described as necessary, and not get too much into details. Average reader will have enough on his plate even without this distinction — almost noone reading this will have any prior experience with Rust, I suppose...

Your idea of splitting `#[eth_abi]` macro into two, one for Endpoint and another for Client — that one I support wholeheartedly.